tox.ini

```diff
@@ -156,6 +156,7 @@ commands_pre =
     bash -c "set -e && cd {toxinidir}/examples/ase && bash train.sh"
     bash -c "set -e && cd {toxinidir}/examples/programmatic/llpr && bash train.sh"
     bash -c "set -e && cd {toxinidir}/examples/zbl && bash train.sh"
+commands =
```

Comment:
We can even put everything in commands IMO, and/or move the bash things above to `tests` unless they are required to run for the docs

Reply:
Agree, the only thing that is not required here is the first line. I would keep it here for consistency.